packages/@cdktf/cli-core/src/test/lib/terraform-provider-lock.test.ts

```diff
@@ -1,10 +1,11 @@
 import { TerraformProviderLock } from "../../lib/terraform-provider-lock";
-import { readFile } from "fs/promises";
+import { readFile, stat } from "fs/promises";
 import { ProviderConstraint } from "../../lib/dependencies/dependency-manager";
 
 jest.mock("fs/promises", () => {
   return {
     readFile: jest.fn().mockResolvedValue("contents"),
+    stat: jest.fn().mockResolvedValue({}),
```

Comment:
Since watching The Good Doctor (recommended!) "stat" has a completely new meaning to me 😄
<img src="https://media3.giphy.com/media/zBh7IffRKjbtTbezRn/giphy.gif"/>